common/src/main/java/org/apache/uniffle/common/config/RssBaseConf.java

```diff
@@ -286,6 +286,16 @@ public class RssBaseConf extends RssConf {
           .defaultValue(WritableSerializer.class.getName())
           .withDescription("Serializations are used for creative Serializers and Deserializers");
 
+  public static final ConfigOption<String> REST_AUTHORIZATION_CREDENTIALS =
+      ConfigOptions.key("rss.rest.authorization.credentials")
```

Comment:
Should we use `http` instead of rest?
Should we support other authorization except for basic?
`rss.http.basic.authorizationCrentilas` may be better.

Reply:
Done